en/deploy-on-gcp-gke.md

```diff
@@ -242,6 +242,10 @@ After `terraform apply` is successful executed, perform the following steps to a
     > You need to install the MySQL client before you connect to TiDB via MySQL. If you use CentOS, install the client by executing `sudo yum install -y mysql`.
     > `${tidb_ilb_ip} is the IP address of the Internal LoadBalancer acquired in step 1.
 
+> **Note:**
```

Comment:
Could you put this **Note** somewhere else if possible? It looks weird that two notes are so close to each other.

Reply:
Good catch, I will find a new place to put it.